### PR TITLE
Updated the installation instructions for Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**
It's great to see how simple things are now (for example, there's nothing to say or explain about permissions because you can't have permissions problems when installing Symfony, because of the use of `umask()` by default).

### CODE REVIEW

File: setup.rst [hunk 1]
```diff
@@ -223,68 +124,64 @@ complexity of your project):
...
-        $ php bin/console security:check
+When working collaboratively in a Symfony application, it's uncommon to create
+a new Symfony application as explained in the previous sections. Instead,
+someone else has already created and submitted it to a shared repository.
 
-    A good security practice is to execute this command regularly to be able to
-    update or replace compromised dependencies as soon as possible.
+It's recommended to not submit some files (``.env``) and directories (``vendor/``,
+cache, logs) to the repository, so you'll have to do the following when
+installing an existing Symfony application:
```
Comment (xabbuh) [inline setup.rst]: this is now missing an explanation about what to do with the `.env` file

Comment (javiereguiluz) [inline setup.rst]: Yes ... we'll do that in the upcoming reference article about configuring Symfony apps: #7386

File: setup.rst [hunk 2]
```diff
@@ -4,219 +4,117 @@
...
-    http://localhost:8000/config.php
+The ``req-checker`` utility adds two PHP scripts in your application:
+``bin/check.php`` and ``public/check.php``. Run the first one in the command
+console and the second one in the browser. This is needed because PHP can define
+a different configuration for both the command console and the web server, so
+you need to check both.
 
-Fixing Permissions Problems
----------------------------
+Once you fix all the reported issues, uninstall the requirements checker:
```
Comment (hhamon) [inline setup.rst]: Once you've fixed

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request closed without merging**